numpy/lib/arraysetops.py

```diff
@@ -356,8 +356,10 @@ def intersect1d(ar1, ar2, assume_unique=False, return_indices=False):
     ar1, ar2 : array_like
         Input arrays. Will be flattened if not already 1D.
     assume_unique : bool
-        If True, the input arrays are both assumed to be unique, which
-        can speed up the calculation.  Default is False.
+        If True, the input arrays are both assumed to be unique, which can speed
+        up the calculation.  If True but non-unique arrays are given, incorrect
+        results and out-of-bounds indices could result.  Default is False.
```

Comment:
```suggestion
        If True, the input arrays are both assumed to be unique, which
        can speed up the calculation.  If True but ``ar1`` or ``ar2`` are not
        unique, incorrect results and out-of-bounds indices could result.
        Default is False.
```
Feel free to take-or-leave the wording suggestion. Either way, the line width should be kept under 80 characters.